connector-templates.json

```diff
@@ -733,8 +764,20 @@
     }
   } ],
   "io.camunda.connectors.HttpJson.v2" : [ {
+    "version" : 10,
+    "ref" : "https://raw.githubusercontent.com/camunda/connectors/d1165e0dab196f38d10671180865ff64e08663f3/connectors/http/rest/element-templates/http-json-connector.json",
+    "engine" : {
+      "camunda" : "^8.3"
+    }
+  }, {
+    "version" : 9,
+    "ref" : "https://raw.githubusercontent.com/camunda/connectors/8b1915903760c3a93d496cb7490431f48ec57076/connectors/http/rest/element-templates/http-json-connector.json",
+    "engine" : {
+      "camunda" : "^8.7"
+    }
+  }, {
```

Comment:
@mathias-vandaele Why is version 10 compatible with 8.3 and version 9 with 8.7? Can you check?